### ISSUE TITLE
Admin page error 500

### ISSUE THREAD

**Issue by bjornpijnacker** [go-gitea/gitea#18795]
### Gitea Version

1.16.0

### Git Version

2.35.1

### Operating System

Arch Linux

### How are you running Gitea?

Using the provided docker-compose file

### Database

SQLite

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

https://gist.github.com/bjornpijnacker/1abe0b7f0a6299ec88b3f60ecb63ab2f

### Description

Going to the admin panel yields a 500 error. The logs do not provide much debug information, so I cannot trace whether this is a configuration issue or a Gitea bug.

### Screenshots

![2022-02-17-13:05:38](https://user-images.githubusercontent.com/35508080/154478403-ada92bef-79dc-433b-a8af-c35fb7706a3f.png)

**Comment by kjuulh** [comment]
on version 1.16, also arch linux. Error I am getting is:

```
...s/context/context.go:206:HTML() [E] Render failed: template: admin/dashboard:8:90: executing "admin/dashboard" at <Str2html>: invalid value; expected string
```

**Comment by bjornpijnacker** [comment]
Seems fixed in 1.16.1, yes.
